docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.8"
+services:
+
+  my-custom-springboot-app:
```

Comment:
Can your application access the files from within the docker container? Take a look at docker volume mapping

Reply:
No, we can't. I'll take a look at docker volume mapping.